operator/data/profiles/default.yaml

```diff
@@ -504,6 +504,9 @@ spec:
           - port: 15011
             targetPort: 15011
             name: tcp-pilot-grpc-tls
+          - port: 15012
+            targetPort: 15012
+            name: http-istiod
```

Comment:
should this be `grpc-istiod`? Or tcp

Reply:
I assume tcp, it is used everywhere for port 15012

Reply:
Changed to `tcp-istiod`